modules/node-shared/src/main/scala/io/constellationnetwork/node/shared/infrastructure/snapshot/CurrencySnapshotCreator.scala

```diff
@@ -130,11 +132,16 @@ object CurrencySnapshotCreator {
             _.accept(maybeLastDataApplication, dataBlocks, lastArtifact.ordinal, currentOrdinal)
           )
 
-          (acceptanceResult, messagesAcceptanceResult, acceptedRewardTxs, snapshotInfo, stateProof) <-
+          sharedArtifactsForAcceptance = dataApplicationAcceptanceResult
+            .flatMap(_.sharedArtifacts)
+            .orElse(sharedArtifacts)
```

Comment:
do we plan passing additional artifacts? I think, for now it should be taken from data application only. otherwise empty

Reply:
👍🏼